symfony/client/src/orangehrmAdminPlugin/pages/systemUser/SaveSystemUser.vue

```diff
@@ -129,6 +130,7 @@ export default {
             (v && v.trim().length >= 5) || 'Should have at least 5 characters',
```

Comment:
Let's refactor these with validation functions. `shouldNotExceedCharLength()`

Reply:
_Should have at least 5 characters_
You can add a validation rule for this, if you have spare time.

Reply:
done